lib/builder/ProjectBuildContext.js

```diff
@@ -37,6 +37,14 @@ class ProjectBuildContext {
 		return this._project === this._buildContext.getRootProject();
 	}
 
+	getOptions() {
+		return this._buildContext.getOptions();
```

Comment:
currently, `getOptions` is not used anywhere in our implementation. Also not in the build context. So I tend to remove them. @matz3 your opinion on that?

Reply:
Agree. Let's for now just add `getOption`.

Reply:
Done